package/lib-tensorflow_cc-shared-1.4.0-cpu/patch.linux/build_tensorflow.sh.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/tensorflow_cc/cmake/build_tensorflow.sh b/tensorflow_cc/cmake/build_tensorflow.sh
+index 9957639..7691a46 100755
+--- a/tensorflow_cc/cmake/build_tensorflow.sh
++++ b/tensorflow_cc/cmake/build_tensorflow.sh
+@@ -16,27 +16,11 @@ export TF_CUDA_CLANG=0
+ export PYTHON_BIN_PATH="$(which python3)"
+ export PYTHON_LIB_PATH="$($PYTHON_BIN_PATH -c 'import site; print(site.getsitepackages()[0])')"
+ 
+-# configure cuda environmental variables
+-if [ -e /opt/cuda ]; then
+-    echo "CUDA support enabled"
+-    cuda_config_opts="--config=cuda"
+-    export TF_NEED_CUDA=1
+-    export TF_CUDA_COMPUTE_CAPABILITIES="3.5,5.2,6.1,6.2"
+-    export CUDA_TOOLKIT_PATH=/opt/cuda
+-    export CUDNN_INSTALL_PATH=/opt/cuda
+-    export TF_CUDA_VERSION="$($CUDA_TOOLKIT_PATH/bin/nvcc --version | sed -n 's/^.*release \(.*\),.*/\1/p')"
+-    export TF_CUDNN_VERSION="$(sed -n 's/^#define CUDNN_MAJOR\s*\(.*\).*/\1/p' $CUDNN_INSTALL_PATH/include/cudnn.h)"
+-    # use gcc-5 for now, clang in the future
+-    export GCC_HOST_COMPILER_PATH=/usr/bin/gcc-6
```

Comment:
I'm bit confused about path to `gcc-6` here, while the comment about says `# use gcc-5 for now`. Or does this patch actually remove this?